### PR TITLE
WIP: Implement exercise change

### PR DESCRIPTION
**Pull request by justani98**

### CODE REVIEW

**Comment by justani98** [comment]
Implemented the exercise **change**. This is my first pull request on Github. Can someone please review my implementation and give me required suggestions. Thanks.

**Comment by justani98** [comment]
Hey, Have I done something wrong here? Is this exercise not necessary for the python track? If so, is there any other way for me to contribute. Thanks.

File: config.json [hunk 1]
```diff
@@ -565,6 +565,15 @@
         "strings",
         "pattern matching"
       ]
+    },
+    {
```
Comment (behrtam) [inline config.json]: You might need to `rebase` and add the new metadata: https://github.com/exercism/meta/issues/16

File: exercises/change/example.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+def find_minimum_coins(total_change, coins):
+    if total_change < 0:
+        return -1
+    min_coins_required = [1e9] * (total_change + 1)
+    last_coin = [0]*(total_change + 1)
```
Comment (behrtam) [inline exercises/change/example.py]: I personally would add white spaces around `*` to make it more readable.

File: exercises/change/example.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+                    final_result = result
+                    last_coin[change] = change - coin
+        min_coins_required[change] = final_result
+    if min_coins_required[total_change] == 1e9:
+        return -1
+    else:
+        last_coin_value = total_change
+        array = []
+        while(last_coin[last_coin_value] != -1):
+            array.append(last_coin_value-last_coin[last_coin_value])
```
Comment (behrtam) [inline exercises/change/example.py]: I personally would add white spaces around `-` to make it more readable.

File: config.json (hunk 1 above)
Comment (justani98) [inline config.json]: done

File: exercises/change/example.py (hunk 2 above)
Comment (justani98) [inline exercises/change/example.py]: I will take take care of the white spaces next time. :)

**Pull request merged**